tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,61 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+		const ipv4HeaderSize = 20
+		const icmpHeaderSize = 8
+		const ipv4IcmpHeaderSize = ipv4HeaderSize + icmpHeaderSize
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
+			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
+
+			By("Checking connectivity via ICMP")
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
```

Comment:
default is "-c1 -w5" , -c 1 is enough ? (on other lines u used -c5 , maybe its better?)
should we change in other PR the default of PingFromVMConsole

Reply:
there is one PingFromVMConsole that still use the default "-c 1"
are you fine with it or want to change as you do on the other uses of PingFromVMConsole in this PR please ?

Reply:
Are you talking about the ping with `-s` defining the size of the payload?

Reply:
nope
about 
 "-c1 -w5" , about the c - count packages, and the w (deadline) which should come together in case you want that C packets, and exactly C, must success, this is why the w is needed with c.

Reply:
There is no `"-c1 -w5"` in my PR.

Reply:
is the default of PingFromVMConsole in case you dont give other parameters,
there is one place where you dont give parameters

Reply:
So your comment is for the library that is already merged and I'm just using it.

@EdDev do you consider it a bug in the ping helper implementation?

Reply:
you can still give it custom vars in case the helper wont change, as you did on the 2nd call of this helper

Reply:
I just want to test whether there is connectivity. I expect the helper to do exactly that by default. If it does not do that, I'd consider it a bug in the helper. Working around it by overriding the defaults with our custom attributes would defy the purpose of those defaults.

Reply:
This discussion is hard to follow. Why are you talking about arguments and not the logic itself?
The default logic says it sends one ping and timeout after 5sec if no response arrives.
I think that if not all packets are acked, then the command will still fail (makes sense to me), so I see no reason to any other default.

You need to explain why you think the default needs to be changed (and reasoning that it is done in other places is not good enough) for this specific usage.

Reply:
> This discussion is hard to follow. Why are you talking about arguments and not the logic itself?
> The default logic says it sends one ping and timeout after 5sec if no response arrives.
> I think that if not all packets are acked, then the command will still fail (makes sense to me), so I see no reason to any other default.

as long as we use -c with -w , it indeed fail if any of the packet fails, 
i think we just suggest that the -c default will be 3 - 10 packets and not just one (-w can still be 5s untouched), even a machine that reboots might have a ping for few packets and then lost it in case of some operation it does in the boot (it happens, i saw it), a few more packets can be safer / more realistic test

> 
> You need to explain why you think the default needs to be changed (and reasoning that it is done in other places is not good enough) for this specific usage.